src/ucp/wireup/wireup_cm.c

```diff
@@ -1288,9 +1288,10 @@ ucp_request_t* ucp_ep_cm_close_request_get(ucp_ep_h ep, const ucp_request_param_
         return NULL;
     }
 
-    request->status  = UCS_OK;
-    request->flags   = 0;
-    request->send.ep = ep;
+    request->status               = UCS_OK;
+    request->flags                = 0;
+    request->send.req_id.local   = UCP_REQUEST_ID_INVALID;
+    request->send.ep              = ep;
```

Comment:
1. weird indent
2. maybe add common required initialized to set all those 4 fields